core/src/main/java/org/elasticsearch/search/SearchHit.java

```diff
@@ -797,6 +833,10 @@ public int hashCode() {
      */
     public static final class NestedIdentity implements Writeable, ToXContent {
 
+        public static final String _NESTED = "_nested";
+        public static final String _NESTED_FIELD = "field";
```

Comment:
I think I don't understand. Whats the offset here? Might be my limited knowledge of what these fields represent